### DISCUSSION TITLE
can anybody tell me know about this i got many trouble tp setup foundry in windows while using ternminals

### DISCUSSION THREAD

**Discussion by audit-bykhizar** [Cyfrin/foundry-full-course-cu#2886] [Q&A]
![image_2024-11-20_003943547](https://github.com/user-attachments/assets/449873c7-fb9c-43e4-ae93-98bf76767634)

**Comment by audit-bykhizar** [comment]
![image_2024-11-20_005006119](https://github.com/user-attachments/assets/44f28e85-520d-47f9-9323-c9e5d1c27bf2)
every time i got this error plese help me

**Reply by chainflash** [reply]
first install wsl in terminal windows :
```
wsl --install
```
Then open Ubuntu if it is installed in your system and enter the following commands in Ubuntu :
```
curl -L https://foundry.paradigm.xyz | bash
```
```
foundryup
```
Now, to make sure that Foundry is installed correctly, enter the following command in Ubuntu :
```
forge --version
```
If you do not receive an error, then it is installed correctly 👍

**Reply by audit-bykhizar** [reply]
Thanks, bro! What should I do next?
![Uploading image_2024-11-20_090654265.png…]()

**Reply by audit-bykhizar** [reply]
Bro, can you do me a favor? If I give you access to my PC through Anydesk, can you set up Foundry for me? I’ve been trying for a long time, but I just can’t figure it out. Please help me, I’m not understanding anything.

**Reply by EngrPips** [reply]
Tough.

**Reply by chainflash** [reply]
Ok no problem send me your email Then we use Anydesk.
I will try to fix it 👍

**Reply by EngrPips** [reply]
Awesome.

**Reply by chainflash** [reply]
Keep moving forward 💯

**Reply by audit-bykhizar** [reply]
@chainflash  thanks bro this is my anydesk adress 
630 763 530

**Comment by audit-bykhizar** [comment]
@chainflash "Let's set a specific time, and when both of us are online, then I will accept the AnyDesk request 
btw ( khixarhayat756@gmail.com)  this is my email so when are you free just mail me thn we fix it     thanks a lot bro ❤️❤️❤️

**Reply by chainflash** [reply]
I sent you an email, check it bro

**Comment by EngrPips** [comment]
Awesome community

**Reply by audit-bykhizar** [reply]
yeah We all want to learn this and build the healthy community

**Reply by EngrPips** [reply]
Indeed.

**Reply by chainflash** [reply]
cool 👍

**Comment by audit-bykhizar** [comment]
@EngrPips 
Bro, as you know, I have just started an intermediate course. So, I noticed something in the VS Code terminal. For example, in Patrick's terminal, I see [foundry-23] displayed, but in my VS Code terminal, nothing like that is showing up
![Screenshot 2024-11-23 095611](https://github.com/user-attachments/assets/1b5a3b05-6e17-4d2e-8749-cc56534ae075)
![Screenshot 2024-11-23 095557](https://github.com/user-attachments/assets/7dfb5062-fc2a-4fcb-8987-ac20bce74c8f)

**Reply by audit-bykhizar** [reply]
@EngrPips ?

**Reply by EngrPips** [reply]
I will suggest you read this post  https://superuser.com/questions/60555/show-only-current-directory-name-not-full-path-on-bash-prompt on how to achieve what you pointed out but I don't think that is really a concern.

**Comment by audit-bykhizar** [comment]
![Uploading molaa.png…]()
After setting up VS Code, how can we delete the default files we get and reset or renew them? I haven’t started coding yet, but I think some mistake I made caused an error, so I want to set it up again.

**Reply by EngrPips** [reply]
Your image is not loading for me. But if you right-click on the files you want to delete you should see the delete option

**Comment by audit-bykhizar** [comment]
And how i will start against

**Comment by audit-bykhizar** [comment]
Like by using just vs coad terminal or connects with gitpod remote

**Comment by audit-bykhizar** [comment]
And can we work without using gitpod or.  We can only VS code  instead if
connected with gitpod

**Reply by EngrPips** [reply]
Honestly, I have never used gitpod as I work on Mac. So, I have little knowledge about how gitpod works.
